server/routes/serviceProviderReferrals/reviewActionPlanPresenter.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ActionPlan,
+  Activity,
+  DesiredOutcome,
+  SentReferral,
+  ServiceCategory,
+} from '../../services/interventionsService'
+import utils from '../../utils/utils'
+
+export default class ReviewActionPlanPresenter {
+  constructor(
+    private readonly sentReferral: SentReferral,
+    private readonly serviceCategory: ServiceCategory,
+    private readonly actionPlan: ActionPlan
+  ) {}
+
+  readonly submitFormAction = `/service-provider/action-plan/${this.actionPlan.id}/submit`
+
+  private readonly desiredOutcomesIds = this.sentReferral.referral.desiredOutcomesIds
+
+  readonly text = {
+    title: `${utils.convertToProperCase(this.serviceCategory.name)} - create action plan`,
+    subTitle: `Review ${this.sentReferral.referral.serviceUser.firstName}’s action plan`,
+    numberOfSessions: this.actionPlan.numberOfSessions?.toString() ?? '',
```

Comment:
Is the number of sessions optional, or is this because we don't have the number of sessions page yet?

Reply:
It's nullable on the `ActionPlan` model. In the actual journey it will have been set by this point, but since the type's nullable we still need to handle null or the compiler will whine.